### PR TITLE
Add TidierData to frameworks docs page

### PR DESCRIPTION
**Pull request by drizk1**
This PR adds TidierData to the frameworks manipulations docs page. 

I tried to keep it brief and similar to the other framework examples. 

I am happy to make any changes. 

@kdpsingh @bkamins 

Thank you

### CODE REVIEW

**Comment by kdpsingh** [comment]
@drizk1, thanks so much for drafting this. I'll make a few editorial tweaks soon before it's ready for a look by maintainers.

**Comment by bkamins** [comment]
Fix https://github.com/JuliaData/DataFrames.jl/issues/3446

**Comment by bkamins** [comment]
@kdpsingh - I will review and merge it after your approval. Thank you!

**Comment by kdpsingh** [comment]
Thanks, I'll take a look in the next 2-3 days!

**Comment by kdpsingh** [comment]
Sorry for the delay - returning to this soon.

**Comment by drizk1** [comment]
@bkamins i have added @kdpsingh edits.

It is ready for your review 

Thank you

**Comment by kdpsingh** [comment]
Thanks @drizk1. Appreciate your working with me offline. Tried to directly commit my changes to this PR but I am still a bit of a newbie.

File: docs/src/man/querying_frameworks.md
Comment (bkamins) [inline docs/src/man/querying_frameworks.md:14]:
```suggestion
data analysis interface that wraps DataFrames.jl.  The instructions below are for version 
```

File: docs/src/man/querying_frameworks.md
Comment (bkamins) [inline docs/src/man/querying_frameworks.md:30]:
```suggestion
data frames. TidierData re-exports `DataFrame` from DataFrames.jl, `@chain` from Chain.jl, and 
```

**Review by bkamins — APPROVED** [review]

**Comment by bkamins** [comment]
Thank you!

**Pull request merged**
